signal_interpolation_fourier.py

```diff
@@ -0,0 +1,589 @@
+# Module for Fourier interpolation of pulsed signals along simulated radio footprints of cosmic-ray air showers
+# Author: A. Corstanje (a.corstanje@astro.ru.nl), 2023
+#
+# See article: arXiv:xxxx.xxxxx, doi xxx
+# Please cite this when using code and/or methods in your analysis
+
+import numpy as np
+from scipy.signal import hilbert, resample
+import interpolation_fourier as interpF
+
+class interp2d_signal:
+
+    def get_spectra(self, signals, verbose=False):
+        """
+        # Do FFT of 'signals', assumed shape (Nants, Nsamples, Npol) i.e. the time traces are along the second axis.
```

Comment:
Have you missed the parameter section? Or just given up? ;)

Reply:
Well when there is just one parameter and I have just described what it is... 
It's strict vs casual; if you are strict it needs to have a parameter section even when there's no parameter.